pyproject.toml

```diff
@@ -1,14 +1,70 @@
 [build-system]
 requires = [
-  "setuptools >= 42.0.0",  # required by pyproject+setuptools_scm integration
-  "setuptools_scm[toml] >= 3.5.0",  # required for "no-local-version" scheme
-  "setuptools_scm_git_archive >= 1.0",
-  "wheel",
+  "setuptools_scm[toml] >= 7.0.0",
```

Comment:
You need "setuptools >= 61.0` here.

Reply:
Right... I wronly assumed the they would have bumped it but I see https://github.com/pypa/setuptools_scm/blob/main/setup.cfg#L32